backends-velox/src/test/scala/io/glutenproject/execution/TestOperator.scala

```diff
@@ -365,4 +365,22 @@ class TestOperator extends WholeStageTransformerSuite {
       }
     }
   }
+
+  test("stddev_samp") {
+    withSQLConf("spark.sql.adaptive.enabled" -> "false") {
```

Comment:
Whether Stddev not support AQE?

Reply:
This is due to an unresolved common issue I found in plan check. In the unit tests, if AQE is enabled, in some cases `transformer` cannot be found through the executed plan.  So in this file some unit tests were marked as AQE disabled for now.